matrix/buffer.py

```diff
@@ -685,10 +718,29 @@ def _membership_message(self, user, message_type):
 
         return message
 
+    def _nick_change_message(self, user, old_nick, new_nick):
+        action_color = "purple"
+        prefix = "network"
+
+        message = (
```

Comment:
This doesn't print out the disambiguated nicks which can lead to confusion when we have duplicates, maybe it'll make sense to always print out the user id alongside the first nick?

Also if you want to color the message, please make it configurable. The irc nick change message uses the default color, I would stick to that as a default as well.